config_proc_macro/src/attrs.rs

```diff
@@ -39,10 +39,7 @@ pub fn is_config_value(attr: &syn::Attribute) -> bool {
 }
 
 fn is_attr_name_value(attr: &syn::Attribute, name: &str) -> bool {
-    attr.parse_meta().ok().map_or(false, |meta| match meta {
-        syn::Meta::NameValue(syn::MetaNameValue { ref path, .. }) if path.is_ident(name) => true,
-        _ => false,
-    })
+    attr.parse_meta().ok().map_or(false, |meta| matches!(meta, syn::Meta::NameValue(syn::MetaNameValue { ref path, .. }) if path.is_ident(name)))
```

Comment:
This is not being formatted due to the chain length issue and would need to be hand formatted. We can address this later though